.idea/jarRepositories.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is not relevant to the bug fix. You need to revert the addition of this file.

Reply:
Note that when this PR https://github.com/openjdk/jfx/pull/1009 is merged, this file is inside the `.gitignore` and mistakes like this can not happen anymore.